ocfweb/static/scss/pages/_tv.scss

```diff
@@ -9,10 +9,12 @@ $tv-hours-small: 5;
     background-repeat: no-repeat;
     background-size: cover;
     height: 100vh;
+    margin-top: -60px;
```

Comment:
You'll want to use the variable `$navbar-height` rather than hardcoding its value.